Jamulus.pro

```diff
@@ -25,6 +25,16 @@ CONFIG += qt \
     lrelease \
     release
 
+CONFIG (noportaudio) {
+    CONFIG -= portaudio
+}
```

Comment:
Any specific reason for this? I don't think we have `no*` CONFIG values for everything else either, probably just in those places where we need to override defaults.

Reply:
Oh, that was leftover from when I had portaudio as the default. Dropping it.